docs/source/project_documentation/user_guide/model_definitions.md

```diff
@@ -60,8 +60,20 @@ triggers (`when`), and post-conditions (`then`) for the behavior.
     :lines: 6-38
     :emphasize-lines: 19, 21, 23, 25, 28, 30, 32
 ```
+### External Models
+
+An `External Model` is a `model` that does not include an `Acceptance Scenario`.  These types of `model`s are used to create simplified interfaces representing external components or actors.  A typical use case for `External Model` is to represent a user or external messenger that is not managed by `AaC`.  It allows you to quickly create `model`s without having to fully implement them.
```

Comment:
Or put it in regular quotes. Because correct, they are not referenced within the YAML models as `External Model` or `Acceptance Scenario`. It's still just a `model` and would be `acceptance`. 

Good catch.